infrastructure/transform/terragrunt.hcl

```diff
@@ -23,5 +23,7 @@ dependency "core" {
 inputs = {
   core_rg_name     = dependency.core.outputs.core_rg_name
   core_rg_location = dependency.core.outputs.core_rg_location
-  spark_version    = get_env("SPARK_VERSION", "3.3.1") // This only needs a default for CICD, which can be removed following https://github.com/UCLH-Foundry/FlowEHR/issues/42 
+  spark_version    = get_env("SPARK_VERSION", "3.3.1") // This only needs a default for CICD, which can be removed following https://github.com/UCLH-Foundry/FlowEHR/issues/42
+  whl_file_local_path  = "../../transform/features/dist"
```

Comment:
I think it's fine to make these whl_file_local_path and whl_file_name a local, I don't think we need to provide it as an input to this module